File: src/nodes/var_const_call.zig
pub const NodeVarConstCall = struct {
    name: []const u8,

    pub fn init(name: []const u8) NodeVarConstCall {
        return NodeVarConstCall{ .name = name };
    }

    pub fn get_name(self: *const NodeVarConstCall) []const u8 {
        return self.name;
    }
};

File: src/nodes/var_const_call.rs
pub struct NodeVarConstCall {
    name: String,
}

impl NodeVarConstCall {
    pub fn init(name: String) -> NodeVarConstCall {
        NodeVarConstCall { name }
    }
    
    pub fn get_name(&self) -> &String {
        &self.name
    }
}